tests/unit/scripts/test_kg_build.py

```
@@ -0,0 +1,445 @@
+    llm = kg.SharedOpenAILLM(Client(), settings)
+    res = asyncio.run(llm.ainvoke("asynchronous"))
+    assert res.content == "ACK"
+
+
+def test_collect_counts_handles_neo4j_error(monkeypatch: pytest.MonkeyPatch) -> None:
+    class DummyNeo4jError(Exception):
+        pass
+    monkeypatch.setattr(kg, "Neo4jError", DummyNeo4jError)
+
+    class Driver:
+        def execute_query(self, query: str, *, database_: str | None = None):
```

Ruff (0.13.1)

Unused method argument: `database_`

(ARG002)